packages/errors/src/errors.ts

```diff
@@ -351,6 +351,30 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/run-group-name-not-unique`
   },
+  CLOUD_AUTO_CANCEL_NOT_AVAILABLE_IN_PLAN: (arg1: {link: string}) => {
+    return errTemplate`\
+      ${fmt.highlightSecondary(`Auto Cancellation`)} is not included under your current billing plan.
+      To enable this service, please visit your billing and upgrade to another plan with Auto Cancellation.
+      ${fmt.off(arg1.link)}`
+  },
+  CLOUD_AUTO_CANCEL_MISMATCH: (arg1: {runUrl: string}) => {
+    return errTemplate`\
+        You passed the ${fmt.flag(`--auto-cancel-after-failures`)} flag, but this run originally started with a different value on this --auto-cancel-after-failures flag.
```

Comment:
What was the original value?

Reply:
Updated it here [63a7744](https://github.com/cypress-io/cypress/pull/25237/commits/63a77440084a5d0b210fa410950b259bdf6e3007)